### PR TITLE
Fixes #1089:Updated Project's Readme

### LINKED ISSUE #1089: Update Project's Documentation in Readme file
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Expected behavior:**
So many changes have been made to the project for which documentation must be made in Readme.md file.
**Screenshots of the issue:**

### PR DESCRIPTION
**Pull request by praveenojha33**
Fixes #1089 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:
Updated Project's Readme
Surge Link: https://pr-1090-fossasia-susper.surge.sh

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +35,7 @@ Search results are displayed using Solr server which is embedded into YaCy. All
 
 ## Requirements
 * [Angular-cli](https://github.com/angular/angular-cli#installation)
-* node --version >= 6
+* node --version >= 8.2.1
 * npm --version >= 3
```
Comment (simsausaurabh) [inline README.md]: @praveenojha33 Would npm version v3 or related work with node v8.* ?

Comment (praveenojha33) [reply README.md]: Since Project works fine with node v6. Should I live it as it is.

Comment (simsausaurabh) [reply README.md]: Keep the minimum working version in README. Since it works for v6.*, it should be kept as it is.

Comment (praveenojha33) [reply README.md]: Done

**Pull request merged**